lib/context.js

```diff
@@ -379,37 +380,37 @@ api.createTermDefinition = (activeCtx, localCtx, term, defined) => {
   }
 
   if('@container' in value) {
-    // Normalize container to an array form
+    // normalize container to an array form
     const container = _isString(value['@container']) ? [value['@container']] : (value['@container'] || []);
-    const validContainers = ['@list', '@set', '@index', '@language'];
+    const validContainers = new Set(['@list', '@set', '@index', '@language']);
     let isValid = true;
     let hasSet = container.includes('@set');
 
     // JSON-LD 1.1 support
     if(activeCtx.processingMode && activeCtx.processingMode.indexOf('json-ld-1.1') === 0) {
       // TODO: @id and @type
-      validContainers.push('@graph');
+      validContainers.add('@graph');
 
       // check container length
       isValid &= container.length <= (hasSet ? 2 : 1);
     } else {
-      // container must be a string which is one of the validContainers
+      // in JSON-LD 1.0, container must not be an array (it must be a string, which is one of the validContainers)
       isValid &= !_isArray(value['@container']);
 
       // check container length
       isValid &= container.length <= 1;
     }
 
-    // Check against valid containers
-    isValid &= container.every(c => validContainers.includes(c));
+    // check against valid containers
+    isValid &= container.every(c => validContainers.has(c));
 
     // @set not allowed with @list
     isValid &= !(hasSet && container.includes('@list'));
 
     if(!isValid) {
       throw new JsonLdError(
         'Invalid JSON-LD syntax; @context @container value must be ' +
-        'one of the following: ' + validContainers.join(', '),
+        'one of the following: ' + Array.from(validContainers).join(', '),
```

Comment:
Can use spread operator instead of `Array.from`:

`[...validContainers].join(', '),`